redaxo/src/core/lib/fragment.php

```diff
@@ -6,11 +6,21 @@
 class rex_fragment
 {
     /**
-     * filename of the actual fragmentfile.
+     * name of the fragment.
+     *
+     * filename relative to the fragment dirs.
      *
      * @var string
      */
-    private $filename;
```

Comment:
hier war nicht wirklich ein filename drinnen, sondern immer nur der relative pfad eines fragments zu den jeweiligen fragmentDirs. daher hab ich das bzgl. readability umbenannt in fragmentName um nicht mit dem eigentlichen $filename verwechselt werden zu können (der wirkliche filename innerhalb des filesystem inkl. fragmentDir etc)